### ISSUE TITLE
External links showing as "Broken URLs"

### ISSUE THREAD

**Issue by jacksorjacksor** [patrickdemooij9/SeoToolkit.Umbraco#238]
Hi!

I've just started using SeoToolkit and I've got error messages consistently for broken links. They're all external links and I think there's different reasons behind each, but I figured I'd mention them:

![image](https://github.com/patrickdemooij9/SeoToolkit.Umbraco/assets/5838388/7c8c7708-f53f-4452-985e-7202c5014937)

```
1. Broken url: https://outlook.office365.com/owa/calendar/NightingaleHQ@nightingalehq.ai/bookings/ (302)
2. Broken url: https://www.linkedin.com/company/nightingale-hq/ (999)
3. Broken url: https://twitter.com/nightingalehqai (403)
```

1. is an outlook link to a calendar bookings form - unsure if anything can be done for that to not be an error?
2. is LinkedIn! All links to LinkedIn pages get 999s, so any advice on that would be ideal
3. is Twitter/X, which I'm guessing will always error as you need to be logged in to Twitter/X to access the content there.

Anyway, let me know if there's anything I can do at my end for any of these issues!

Cheers,

Rich

**Comment by patrickdemooij9** [comment]
Hi @jacksorjacksor
Thank you for the report. Those are indeed annoying to see, especially if they don't represent an issue. I'll see if I can add some logic in the codebase to ignore those URLs

**Comment by jacksorjacksor** [comment]
Hi @patrickdemooij9 - ah thank you! 

I, of course, trust your judgement on the best way to handle these sorts of errors. I was considering that having an "exceptions" list which can be manually defined by an administrator to control what URLs should be ignored might be appropriate. I was just checking whether I'd done something wrong at my end!

Thanks again for your prompt response.

**Comment by patrickdemooij9** [comment]
Yeah, I was thinking about something like that as well. And no, you didn't do anything wrong. Some websites just try to block crawlers from checking their pages and unfortunately we'll then see broken links
